.gitignore

```diff
@@ -12,6 +12,7 @@ app
 snprc_scheduler/resources/web/snprc_scheduler/app/app.js
 *.iml
 *packageExport*.snd.xml
+snprc_ehr/.gradle
 snprc_ehr/resources/views/HelloApp*.*
 snprc_ehr/resources/views/helloWorld*.*
 snprc_ehr/resources/views/NewAnimalPage*.*
```

Comment:
should these be replace with ignoring snprc_ehr/resources/views/gen?

Reply:
Added them in.  Thanks.